introduction.md

```diff
@@ -0,0 +1 @@
+I even dont know what to say. I'm fine, you are fine. Take care.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.